update-script.js

```diff
@@ -0,0 +1,141 @@
+const KintoClient = require("kinto-http").default;
+const btoa = require("btoa");
+const fetch = require("node-fetch");
+const AppConstants = require("./app-constants");
+
+const WEBSITES_WITH_SHARED_CREDENTIAL_COLLECTION = "websites-with-shared-credential-backends";
```

Comment:
`COLLECTION_ID` would be shorter for the rest of the script

Reply:
Fixed, makes sense to me